### PR TITLE
Enable Travis CI

### PR DESCRIPTION
**Pull request by henkmollema**
Fix KoreBuild folder name in build.sh too.

### CODE REVIEW

**Comment by henkmollema** [comment]
@anurse it seems to have a problem with `dnvm`/`dotnet`:
>Successfully installed 'KoreBuild-dotnet 0.2.1-rc1-15566' to packages
packages/KoreBuild-dotnet/build/dnvm.sh: line 3: $'\r': command not found
packages/KoreBuild-dotnet/build/dnvm.sh: line 17: $'\r': command not found
packages/KoreBuild-dotnet/build/dnvm.sh: line 36: syntax error near unexpected token `$'{\r''
'
./build.sh: line 51: dnvm: command not found
./build.sh: line 52: dnvm: command not found
packages/KoreBuild-dotnet/build/install.sh: line 6: $'\r': command not found
packages/KoreBuild-dotnet/build/install.sh: line 8: $'\r': command not found
packages/KoreBuild-dotnet/build/install.sh: line 56: $'\r': command not found
packages/KoreBuild-dotnet/build/install.sh: line 60: syntax error near unexpected token `elif'

and:
> info:   program: dotnet
info:   commandline: restore --quiet 
info:   workingdir: /home/travis/build/aspnet/PlatformAbstractions
fail: ApplicationName='dotnet', CommandLine='restore --quiet ', CurrentDirectory='/home/travis/build/aspnet/PlatformAbstractions', Native error= Cannot find the specified file

**Comment by muratg** [comment]
@victorhurdugaci

**Comment by victorhurdugaci** [comment]
Since you're already updating these files, can you please use the `build.cmd` and `build.sh` from this folder: https://github.com/aspnet/Universe/tree/release/build-template-dotnet ?

**Comment by anurse** [comment]
The build failure is somewhat related but a little tricky. I'll look at correcting it and merge this. Thanks!

**Comment by henkmollema** [comment]
@victorhurdugaci I've updated it. The `dotnet` part works now. Still fails as @anurse noted.

**Comment by victorhurdugaci** [comment]
Thanks! Waiting for @anurse to fix that

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+language: csharp
```
Comment (victorhurdugaci) [inline .travis.yml]: Add OS: https://github.com/aspnet/dnx/blob/dev/.travis.yml#L8-L10

**Comment by anurse** [comment]
The '\r\n' issue is caused by https://github.com/aspnet/Universe/pull/336, which should be fixed soon

There is a different issue that I need to resolve around net451 reference assemblies. It should be fairly straightforward but it'll take a bit.

**Comment by anurse** [comment]
Ok, so you're going to hate me. But can you move this back to the latest `dev` branch? (Make sure you pull first). I promise this will be the last time I ask you to move it :)

We changed our mind again and are actually reverting to the old `dnx`-based build system for now (which means `dev` branch). Sorry for the wild goose chase!

**Comment by anurse** [comment]
Actually I'm pretty sure I can do it, but preserve you as the author on the commits. Stand by :)

**Comment by henkmollema** [comment]
Haha, don't worry :smile:

#8

**Comment by anurse** [comment]
Nevermind, you got it :). Thanks!

**Pull request closed without merging**